src/main/java/io/kokuwa/maven/helm/UpgradeMojo.java

```diff
@@ -0,0 +1,45 @@
+package io.kokuwa.maven.helm;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import lombok.Setter;
+
+/**
```

Comment:
Why does it say "simulate"? As far as I understand, it does an acutal upgrade on a connected cluster. But it would be nice to add the "--dry-run" option, so that it can be used for simulating upgrades.

Reply:
the comment was basically a copy from an other mojo. So I changed it now.

The idea with the dry-run flag is a good suggestion, I will add this later today or tomorrow

Reply:
added dry run flag